packages/definitions-parser/src/mocks.ts

```diff
@@ -12,7 +12,7 @@ class DTMock {
       JSON.stringify({
         packages: {
           angular: {
-            libraryName: "Angular 2",
+            libraryName: "angular 2",
```

Comment:
This is also not a valid library name though?

Reply:
Yeah...I think at one point non-npm libraries were allowed here, but checking notNeededPackages, that's not the case. I'll adjust the test.

Reply:
oh, in fact there is now a check that both the types package and the libraryName package exist, so this mock data was just wrong.